### PR TITLE
Update footer links and add download app badge

### PR DESCRIPTION
**Pull request by williardx**
https://artsyproduct.atlassian.net/browse/GRO-129

- Updates the footer to reorganize links and use new links where appropriate.
- Adds a "Download on the app store" badge to the footer that links out to the app store and tracks clicks on this link.

cc @artsy/grow-devs 

## Screenshots

Screen Shot 2021-01-27 at 2 56 06 PM
Screen Shot 2021-01-27 at 2 56 52 PM
Screen Shot 2021-01-27 at 3 53 14 PM

### CODE REVIEW

File: src/v2/Components/DownloadAppBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import { useAnalyticsContext } from "v2/Artsy"
+import Events from "v2/Utils/Events"
+import { Box, Link } from "@artsy/palette"
+
+interface DownloadAppBadgeProps {
+  contextModule: ContextModule
+}
+
+export const DownloadAppBadge: React.FC<DownloadAppBadgeProps> = track(
+  {},
```
Comment (damassi) [inline src/v2/Components/DownloadAppBadge.tsx:13]: (nonblocking) can maybe just pass null here, like in Boot

File: src/v2/Components/__tests__/DownloadAppBadge.jest.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    expect(trackEvent).toHaveBeenCalledWith(
+      expect.objectContaining({
+        context_module: "footer",
+        destination_path:
+          "https://apps.apple.com/us/app/artsy-buy-sell-original-art/id703796080",
+        subject: "Download on the App Store",
+      })
+    )
+  })
+})
```
Comment (damassi) [inline src/v2/Components/__tests__/DownloadAppBadge.jest.tsx:38]: Ty for the tests 👍

**Review by damassi — APPROVED** [review]
One tiny comment, otherwise looks great 👍

**Review by jonallured — APPROVED** [review]
So goooood! 😍

**Comment by williardx** [comment]
#mergeOnGreen

**Pull request merged**